src/transformers/adapters/hub_mixin.py

```diff
@@ -1,3 +1,4 @@
+# docstyle-ignore-file
```

Comment:
Why is this file ignored?

Reply:
I didn't want to have the adapter card template strings reformatted